shared/src/business/entities/Document.js

```diff
@@ -288,6 +318,7 @@ joiValidationDecorator(
       .description(
         'A lodged document is awaiting action by the judge to enact or refuse.',
       ),
+    mailingDate: joi.string().max(100).optional(),
```

Comment:
Should this be validated as a date instead of just a string?

Reply:
I thought the same thing, but this is a plain text field in the UI, by request of the users.